.github/workflows/build.yml

```diff
@@ -16,7 +16,17 @@ jobs:
       - name: Install Dependencies
         run: |
           apt update
-          apt install -y libarchive-dev libgoocanvas-2.0-dev valac meson appstream-util xvfb at-spi2-core
+          apt install -y libarchive-dev valac meson appstream-util xvfb at-spi2-core
+      - name: Dev dependencies
+        - git clone https://gitlab.gnome.org/GNOME/goocanvas
+        - cd goocanvas
+        - git checkout goocanvas-3.0
+        - mkdir build
+        - cd build
+        - ../autogen.sh
+        - ../configure --prefix /usr
+        - make
+        - make install
```

Comment:
goocanvas doesn't have meson support yet? otherwise those should be inside a run, see the install dependencies step

Reply:
Indeed, it does not: https://gitlab.gnome.org/GNOME/goocanvas/-/issues/7

Would you be able to take care of porting goocanvas from make to meson?
Otherwise I can give it a try later next week.

But yeah, indeed this passage needs to be inside a `run: |` action.

Reply:
Let me rework that. I was talking with @inigomartinez to do the migration. Shouldn't be any major problem to do it